www/docs/customization/changelog.md

```diff
@@ -25,6 +25,13 @@ changelog:
   # Default is empty
   sort: asc
 
+  # Group commits by the following criteria:
+  # - 'feat': commits that add new features.
+  # - 'fix': commits that fix existing issues.
+  # - 'other': all other commits.
+  # Default is false
+  group: true
```

Comment:
I'm thinking if it would make sense to make this "configurable", e.g.:

```yaml
groups:
- title: Features
  regexp: "^.*(feat):*.*$"
  order: 0
- title:  'Bug fixes'
  regexp: "^.*(fix):*.*$"
  orde: 1
- title: Others
  order: 999
```

then the user could setup the categories as they wish.


wdyt?

Reply:
@caarlos0 done!